EC minutes for 28 May 2024 (#30)
jupyter-governance/ec-team-compass: docs/meeting_notes/2024.md

# Meeting notes from 2024

## 28 May 2024

### Attendees

- Jason Grout
- Fernando Perez
- Zach Sailer
- Afshin Darian
- Ana Ruvalcaba

### Notes

* Continued work on creating a PR for the EC/SSC voting about moving to Linux Foundation
* Met with Linux Foundation folks to ask questions regarding the Jupyter Foundation and its governing body, as well as to clarify the entities, relationships, and responsibilities of LF, LF Charities, Project Jupyter, and the Jupyter Foundation.

> * We decided (but have not yet voted on) that the EC can and should raise the quorum requirements for EC votes about governance change even if the decision-making guide provides a low quorum and decision-making threshold.

Author: ```suggestion
* We discussed (but have not yet voted on) the EC raising the quorum and approval requirements for EC votes about governance change even if the decision-making guide provides a low quorum and decision-making threshold.
```
